spec.bs

```diff
@@ -2104,6 +2081,19 @@ An auction config is a [=struct=] with the following items:
 
 </dl>
 
+<div algorithm="validate and convert per buyer signals">

```

Comment:
This is now called once, but I think it makes error-handling cleaner to have it separate...